docs/guides/moodleapp/customisation.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Moodle App Customisation
```

Comment:
As these pages are placed under the "Moodle App" section, I think it would be better to remove this part, at least from the title displayed in the sidebar. The same change should be also applied to pre-existing pages.

Reply:
Solved, and done now